CaloMC/fcl/prolog.fcl

```diff
@@ -44,17 +43,29 @@ CaloShowerROMaker :
     diagLevel                : 0
 }
 
+
```

Comment:
Which of these parameters specifies which noise generation algorithm (full, AR, sample) is being used by default?

Reply:
generateSpotNoise = true will use the spot noise technique, otherwise you'll add the full noise for every waveform
enableAR = true  generates a full waveform with the auto-regressive fit if generateSpotNoise =false. if enableAR is false and generateSpotNoise =false, then you generate full noise waveform with simulating all PE (the old method)